### PR TITLE
Added hex placement sound

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

Also changed `PlayCustomSound` in GUICommon to allow concurrent audio players so that the placement sound works nicely.

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Review by RacerBG — APPROVED** [review]
> Also changed PlayCustomSound in GUICommon to allow concurrent audio players so that the placement sound works nicely.

I have no idea what you mean but the new sound works and I like it. :smiley:

**Comment by Kasterisk** [comment]
Basically it means that the sounds can be played on top of each other without interrupting one another.

File: src/gui_common/GUICommon.cs [hunk 1]
```diff
@@ -48,20 +43,9 @@ private GUICommon()
     public bool IsAnyExclusivePopupActive => GetCurrentlyActiveExclusivePopup() != null;
 
     /// <summary>
-    ///   The audio player for UI sound effects.
-    /// </summary>
-    private AudioStreamPlayer AudioSource { get; }
-
-    /// <summary>
-    ///   Second audio player for GUI effects. This is used if the primary one is still playing the previous effect.
+    ///  The audio players for UI sound effects.
```
Comment (hhyyrylainen) [inline src/gui_common/GUICommon.cs:46]: Is this missing one space (so there's 2 instead of 3) on this line?

**Review by hhyyrylainen — CHANGES_REQUESTED** [review]
Just one small style issue I noticed, otherwise looked good.

File: src/gui_common/GUICommon.cs (hunk 1 above)
Comment (Kasterisk) [reply src/gui_common/GUICommon.cs]: Yes, fixed it.

**Review by hhyyrylainen — APPROVED** [review]

**Pull request merged**